_includes/head.html

```diff
@@ -0,0 +1,5 @@
+<head>
+  {% include analytics.html %}
```

Comment:
If this include is now shared, the contents of `analytics.html` could simply be moved inside here again, so devs don't have to walk one extra step to find it.